spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppAdmin.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.spi.app;
+
+import java.util.Map;
+
+import org.springframework.util.StringUtils;
+
+public class AppAdmin {
```

Comment:
A high level description of the purpose of this class may be helpful. It does feel like an internal abstraction as well - if so, might be worth talking about that as well.

Reply:
Let's add in merge.